src/sssom_schema/schema/sssom_schema.yaml

```diff
@@ -657,11 +660,31 @@ classes:
     - object_preprocessing
     - semantic_similarity_score
     - semantic_similarity_measure
+    - similarity_score
+    - similarity_measure
     - see_also
     - issue_tracker_item
     - other
     - comment
     class_uri: owl:Axiom
+    rules:
```

Comment:
This only addresses the constraint on `subject_id / subject_label`. If we want to allow literals on the object side as well, we need another set of rules for `object_id / object_label`.

Reply:
Yes, I wanted some confirmation from some LinkML person (reached out on their slack) to see if this makes sense this way.

Reply:
I asked for a full review from @cmungall so I committed the object rules as well